test/parallel/test-fs-write-stream-autoclose-option.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
+
+var file = path.join(common.tmpDir, 'write-autoclose-opt1.txt');
+common.refreshTmpDir();
+var stream = fs.createWriteStream(file, {flags: 'w+', autoClose: false});
+stream.write('Test1');
+stream.end();
+stream.on('finish', common.mustCall(function() {
+  process.nextTick(function() {
+    assert.strictEqual(!stream.closed, true);
+    assert.strictEqual(!stream.fd, false);
```

Comment:
Just curious, is there a reason to negate first instead of just checking for the expected value?